### PR TITLE
Billing groups info added to MST billing page

### LINKED ISSUE #3380: [Docs RFC] More billing docs for Managed service for TSDB
# Describe change in content, appearance, or functionality

We need to add more information in the timescale docs on the [Managed services for Timescale DB billing doc section](https://docs.timescale.com/mst/latest/billing/).   We need to include documentation on how to create, assign, and manage billing groups in the MST platform.   The information is available in the Aiven docs, and we need to share this same information with our customers.  Here are the links with all the information about billing groups
- https://aiven.io/docs/platform/concepts/billing-groups
- https://aiven.io/docs/platform/howto/create-billing-groups
- https://aiven.io/docs/platform/howto/use-billing-groups
- https://aiven.io/docs/platform/howto/billing-assign-projects

# Subject matter expert (SME)

[If known, who is a good person to ask about this topic]

# Deadline

This should not be a big lift as the content is already created, just needs to be posted.  This is important to our customers.  Due date Aug 21, 2024.  

# Any further info

[Anything else you want to add, or further links]

## Contributing to documentation

We welcome documentation contributions!

*   For information about how to suggest a change, see the [contributing guide](https://github.com/timescale/docs/blob/latest/CONTRIBUTING.md) in our GitHub repository.
*   For information on style and word usage, see the [style guide](https://docs.timescale.com/about/latest/contribute-to-docs)

### PR DESCRIPTION
**Pull request by atovpeko**
# Description

[Short summary of why you created this PR]

# Links

Fixes #[insert issue link, if any]

# Writing help

For information about style and word usage, see the [style guide](https://docs.timescale.com/about/latest/contribute-to-docs/)

# Review checklists

Reviewers: use this section to ensure you have checked everything before approving this PR:

## Subject matter expert (SME) review checklist

*   [ ] Is the content technically accurate?
*   [ ] Is the content complete?
*   [ ] Is the content presented in a logical order?
*   [ ] Does the content use appropriate names for features and products?
*   [ ] Does the content provide relevant links to further information?

## Documentation team review checklist

*   [ ] Is the content free from typos?
*   [ ] Does the content use plain English?
*   [ ] Does the content contain clear sections for concepts, tasks, and references?
*   [ ] Have any images been uploaded to the correct location, and are resolvable?
*   [ ] If the page index was updated, are redirects required
      and have they been implemented?
*   [ ] Have you checked the built version of this content?

### CODE REVIEW

File: mst/billing.md [hunk 1]
```diff
@@ -41,12 +41,91 @@ Migrating a service to another cloud region or different cloud provider does not
...
+and can pay with one saved payment method. 
+
+Billing groups can only be used in one organization. Aiven 
+credits are assigned per billing group and are automatically used to 
+cover charges of any project assigned to that group.
+
+You can track spending by exporting cost information to business 
+intelligence tools using the [invoice API][invoice-api].
+
+To access billing groups in the [Aiven Console][aiven-console], you must be a
```
Comment (billy-the-fish) [inline mst/billing.md]: Careful here, you are using MST not Aiven:

https://portal.managed.timescale.com/account/a4bf28029b59/billing/overview

Billing groups are in:

https://portal.managed.timescale.com/account/a4bf28029b59/billing/billing-groups

File: mst/billing.md [hunk 2]
```diff
@@ -41,12 +41,91 @@ Migrating a service to another cloud region or different cloud provider does not
...
+super admin or account owner.
+
+### Create a billing group
+
+To create a billing group, take the following steps:
+
+1. In the organization, click **Billing** > **Create billing group**.
```
Comment (billy-the-fish) [inline mst/billing.md]: In [MST Console][link to console], click **Billing** > ...

File: mst/billing.md [hunk 3]
```diff
@@ -41,12 +41,91 @@ Migrating a service to another cloud region or different cloud provider does not
...
+1. Enter a name for the billing group and click **Continue**.
+1. Enter the billing details. You can copy these details from another
```
Comment (billy-the-fish) [inline mst/billing.md]: Put the second sentence on a newline so it is under the action please.

File: mst/billing.md [hunk 4]
```diff
@@ -41,12 +41,91 @@ Migrating a service to another cloud region or different cloud provider does not
...
+
+### Create a billing group
+
+To create a billing group, take the following steps:
+
+1. In the organization, click **Billing** > **Create billing group**.
+1. Enter a name for the billing group and click **Continue**.
+1. Enter the billing details. You can copy these details from another 
+   billing group by selecting it from the list. Click **Continue**.
+1. Select the projects to add to this billing group. You can skip this step and
```
Comment (billy-the-fish) [inline mst/billing.md]: Same here, newline for the explanation bit.

File: mst/billing.md [hunk 5]
```diff
@@ -41,12 +41,91 @@ Migrating a service to another cloud region or different cloud provider does not
...
+1. When you have confirmed everything is correct, click **Create**.
+
+### Manage billing groups
+
+To view and update your billing groups, go to the organization and click 
+**Billing**.
+
+- Rename billing groups:
+
+    1. Find the billing group to rename and click **Actions > Rename**.
```
Comment (billy-the-fish) [inline mst/billing.md]: In [MST Console][link to console], bla bla

File: mst/billing.md [hunk 6]
```diff
@@ -41,12 +41,91 @@ Migrating a service to another cloud region or different cloud provider does not
...
+    1. Enter the new name and click **Rename**.
+
+- Update your billing information:
+
+    1. Select the name of the billing group to update.
```
Comment (billy-the-fish) [inline mst/billing.md]: In [MST Console][link to console], click **Billing** > ...  bla bla

File: mst/billing.md [hunk 7]
```diff
@@ -41,12 +41,91 @@ Migrating a service to another cloud region or different cloud provider does not
...
+    1. Click **Edit** to update the details for that section.
+
+- Delete billing groups
+
+    1. Select the name of the billing group to delete.
```
Comment (billy-the-fish) [inline mst/billing.md]: In [MST Console][link to console], click **Billing** > ...  bla bla

File: mst/billing.md [hunk 8]
```diff
@@ -41,12 +41,91 @@ Migrating a service to another cloud region or different cloud provider does not
...
+    1. Click **Actions** next to the current accumulated monthly bill amount.
+    1. Click **Delete** and **Confirm**.
+
+### Assign and unassign projects
+
+To manage projects in billing groups, take the following steps.
+
+- Assign projects to a billing group:
+
+  1. Go to **Billing > Billing groups**.
```
Comment (billy-the-fish) [inline mst/billing.md]: In [MST Console][link to console], click **Billing** > ...  bla bla

File: mst/billing.md [hunk 9]
```diff
@@ -41,12 +41,91 @@ Migrating a service to another cloud region or different cloud provider does not
...
+  1. Click **Cancel** to close the dialog box.
+
+  <Highlight type="note">
+  Assigning a project that is already assigned to another billing group 
+  will unassign it from that billing group.
+  </Highlight>
+
+- Move a project to another billing group
+
+  1. Select the name of the billing group.
```
Comment (billy-the-fish) [inline mst/billing.md]: In [MST Console][link to console], click **Billing** > ...  bla bla

**Review by billy-the-fish — CHANGES_REQUESTED** [review]
Good work, nearly there.

**Review by billy-the-fish — APPROVED** [review]
LGTM. Good doc.

**Comment by ashegrud** [comment]
Hi @atovpeko and @billy-the-fish  , apologies for the late addition here.   

 I need to reopen this as we need to update 1 item from the docs.  Billing > Corporate billing .  It should say  Payment terms for corporate invoices are **14 days net**  .  Instead of Payment terms for corporate invoices are **10 days net**

**Pull request merged**